src/LibraryUtilities.ts

```diff
@@ -249,3 +255,32 @@ export function buildLibraryItemsFromLayoutSpecs(loadedTypes: any, layoutSpecs:
     }
     return convertToLibraryTree(typeListNodes, layoutElements);
 }
+
+function setItemVisible(item: ItemData) {
+    item.visible = true;
+    item.childItems.forEach((childItem) => setItemVisible(childItem));
+}
+
+export function search(text: string, item: ItemData) {
+    let index = item.text.toLowerCase().indexOf(text);
```

Comment:
Shouldn't the above `indexOf` (and also the `for` loop below) to be done only for items that are not `group` type? If we can directly skip these steps for `group` items then it'll save time.